src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/metrics/AllMetrics.java

```diff
@@ -194,9 +204,9 @@ public String toString() {
     }
 
     public static class Constants {
-      public static final String FIELD_DATA_CACHE_NAME = "Field_Data_Cache";
```

Comment:
is there any specific reason to not follow the naming convention ?

Reply:
As discussed, given this is the column field, we will use the lower case naming convention here.